simulation/traffic_simulator/src/entity/ego_entity.cpp

```diff
@@ -60,6 +60,8 @@ auto getVehicleModelType()
 
   if (architecture_type == "tier4/proposal") {
     vehicle_model_type = getParameter<std::string>("vehicle_model_type", "IDEAL_STEER");
+  } else if (architecture_type == "awf/universe") {
```

Comment:
I temporally named "awf/universe". Is there any better name?

Reply:
There is no problem with the name of awf / universe.
I was thinking of using that name as well.